### PR TITLE
155933397 add cancel and links to route view

### PR DESCRIPTION
**Pull request by rckopteri**
# Added
 - Cancel button to sea route wizard
 - Links to wizard steps to make it more usable when editing route

### CODE REVIEW

File: ote/src/cljs/ote/views/route.cljs [hunk 1]
```diff
@@ -64,12 +67,16 @@
...
+     [ui/stepper {:active-step i
+                  :connector (r/as-element [ic/navigation-arrow-forward])}
       (doall
-       (for [{label :label} wizard-steps]
-         ^{:key label}
-         [ui/step
-          [ui/step-label label]]))]
+        (for [{label :label current-step :name} wizard-steps]
+          ^{:key label}
+          [ui/step (when (rc/validate-all-steps route current-step wizard-steps)
```
Comment (tatut) [inline ote/src/cljs/ote/views/route.cljs]: Calculate validity once and put it in the `for` :let, not it is repeated twice: once for style and once for click handler

File: ote/src/cljs/ote/app/controller/route.cljs [hunk 2]
```diff
@@ -260,3 +266,13 @@
                    (every? #(and (time/valid-time? (::transit/departure-time %))
                                  (time/valid-time? (::transit/arrival-time %))) other-stops))))
           trips))
+
+(defn validate-all-steps
```
Comment (tatut) [inline ote/src/cljs/ote/app/controller/route.cljs]: I think a term like "previous" would be better as it does not validate "all" steps.

File: ote/src/cljs/ote/app/controller/route.cljs [hunk 3]
```diff
@@ -229,7 +230,12 @@
   (process-event [{response :response} app]
     (.error js/console "Save route failed:" (pr-str response))
     (assoc app
-           :flash-message-error "Reitin tallennus epäonnistui")))
+      :flash-message-error "Reitin tallennus epäonnistui"))
```
Comment (mmarkuss) [inline ote/src/cljs/ote/app/controller/route.cljs]: Add text to resource file.

Comment (rckopteri) [reply ote/src/cljs/ote/app/controller/route.cljs]: Yes, later.
I think that we can use in this poc only finnish and translate all text at once.

**Pull request merged**